### DISCUSSION TITLE
TaskBee - Task/Query Automation built using PB + Stencil

### DISCUSSION THREAD

**Discussion by polysign** [pocketbase/pocketbase#5204] [Show and tell]
Hi,

in the last few months I build **TaskBee** in my spare time. It's actively being used by 5-10 users, and I tought I'll start doing some polishing and show it to the world.

> https://taskbee.dev

It all started as a replacement app/tool for Airplane.dev, which has been discontinued in March. Instead of building alternatives using No-Code platforms, I decided to jump in and build an _MVP_ using **PocketBase**. It's my first real PB project and I just loved how simple and intuitive I am able to use it.

The logo has been made using ChatGPT :)
I'm looking for some feedback and I will keep improving it over the next couple of months.

**Comment by thangngoc89** [comment]
Just want to let you know that the layout looks completely broken on Safari Version 16.5 (18615.2.9.11.4) - macOS 13.4. I've checked console log, try private mode (no extensions) and there isn't any errors.

<img width="2028" alt="image" src="https://github.com/user-attachments/assets/d363b6d9-9404-4a6d-9cc6-59b63cdcef8e">

**Reply by polysign** [reply]
Hi @thangngoc89. I appreciate your feedback. I will take some time soon to improve the responsiveness of the landing page and fix some browser issues 👍
